specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml

```diff
@@ -28,14 +28,14 @@ options:
     flavor: azure
   "@azure-tools/typespec-ts":
     package-dir: "arm-contoso"
-    azureSdkForJs: true
-    isModularLibrary: true
-    generateMetadata: true
+    azure-sdk-for-js: true
+    is-modular-library: true
```

Comment:
pls upgrade the automation version to verify if `is-modular-library` could work correctly!